skypy/power_spectrum/_camb.py

```diff
@@ -8,13 +14,14 @@
 
 
 def camb(wavenumber, redshift, cosmology, A_s, n_s):
-    """ Return the CAMB computation of the linear matter power spectrum, on a
-    two dimensional grid of wavenumber and redshift
+    r'''CAMB linear matter power spectrum.
+    Return the CAMB computation of the linear matter power spectrum, on a
+    two dimensional grid of wavenumber and , described in [1]_.
 
     Parameters
     ----------
     wavenumber : (nk,) array_like
-        Array of wavenumbers in units of [Mpc^-1] at which to
+        Array of wavenumbers in units of :math:`[Mpc^-1]` at which to
```

Comment:
Does not render properly: <img width="110" alt="image" src="https://user-images.githubusercontent.com/3993688/87304734-2f7a1c80-c50d-11ea-81d4-5ca72ce37573.png">

Reply:
```suggestion
        Array of wavenumbers in units of :math:`[{\rm Mpc}^{-1}]` at which to
```
Perhaps try this? That is what I would do in LaTeX